### PR TITLE
update error codes

### PR DESCRIPTION
**Pull request by hxy7yx**

### CODE REVIEW

File: en_US/configuration/south-devices/modbus-rtu/modbus-rtu.md [hunk 1]
```diff
@@ -82,8 +81,12 @@ Optional, byte order, applicable to data types int16/uint16/int32/uint32/float,
 | #L  | 1,2     | int16/uint16       | Default byte order if not specified |
 | #LL | 1,2,3,4 | int32/uint32/float | Default byte order if not specified |
 | #LB | 2,1,4,3 | int32/uint32/float | |
-| #BB | 3,4,1,2 | int32/uint32/float | |
-| #BL | 4,3,2,1 | int32/uint32/float | |
+| #BL | 3,4,1,2 | int32/uint32/float | |
+| #BB | 4,3,2,1 | int32/uint32/float | |
+
+::: tip
+Byte order may be represented by ABCD, simply correspond 1234 to ABCD. For example, ABCD corresponds to the default Endianness 1234 (# LL).
```
Comment (LenaLenaPan) [inline en_US/configuration/south-devices/modbus-rtu/modbus-rtu.md:88]:
```suggestion
The byte order can be illustrated using the notation ABCD, which corresponds directly to the sequence 1234. As an example, the ABCD designation represents the standard or default Endianness 1234. (# LL).
```

File: en_US/configuration/south-devices/modbus-tcp/modbus-tcp.md [hunk 2]
```diff
@@ -84,8 +84,12 @@ Optional, byte order, applicable to data types int16/uint16/int32/uint32/float,
 | #L  | 1,2     | int16/uint16       | Default byte order if not specified |
 | #LL | 1,2,3,4 | int32/uint32/float | Default byte order if not specified |
 | #LB | 2,1,4,3 | int32/uint32/float | |
-| #BB | 3,4,1,2 | int32/uint32/float | |
-| #BL | 4,3,2,1 | int32/uint32/float | |
+| #BL | 3,4,1,2 | int32/uint32/float | |
+| #BB | 4,3,2,1 | int32/uint32/float | |
+
+::: tip
+Byte order may be represented by ABCD, simply correspond 1234 to ABCD. For example, ABCD corresponds to the default Endianness 1234 (# LL).
```
Comment (LenaLenaPan) [inline en_US/configuration/south-devices/modbus-tcp/modbus-tcp.md:91]:
```suggestion
The byte order can be illustrated using the notation ABCD, which corresponds directly to the sequence 1234. As an example, the ABCD designation represents the standard or default Endianness 1234. (# LL).
```

**Review by LenaLenaPan — APPROVED** [review]

**Comment by LenaLenaPan** [comment]
is this one a 2.6 related feature or can be just merged? @hxy7yx

**Comment by hxy7yx** [comment]
> is this one a 2.6 related feature or can be just merged? @hxy7yx

2.5 and main

**Pull request merged**